src/Componenets/Comments.jsx

```diff
@@ -10,8 +11,22 @@ const Comments = () => {
       })
       .then((data) => {
         setComments(data);
-        console.log(data);
       });
+  }, [reviewId]);
+
+  return comments.map((comment) => {
+    return (
+      <div key={comment.comment_id}>
+        <h2>Comment ID: {comment.comment_id}</h2>
+        <h2>Author: {comment.author}</h2>
+        <h2>Body: {comment.body}</h2>
+        <h2>Created At: {comment.created_at}</h2>
+        <h2>Votes: {comment.votes}</h2>
+        <Link to={`/review/${reviewId}`}>
+          <h2>Click To Go Back To Review</h2>
+        </Link>
+      </div>
+    );
   });
 };
 
```

Comment:
Good use of useParams here. I'd suggest using a useState for loading to keep the user informed about what is happening on slower connections. I'd also suggest using a util function at some point to change the created at into something more user friendly removing the T and Z and seconds.

Really nice functionality with clicking through to comments and back to reviews.

Reply:
cool no worries will add them on and then merge it

Reply:
thank you